src/tokenserver/error.rs

```diff
@@ -78,13 +80,13 @@ impl TokenserverError {
         }
     }
 
-    pub fn unsupported(description: &'static str) -> Self {
+    pub fn unsupported(description: &'static str, name: String) -> Self {
```

Comment:
These changes are unrelated to the original issue (#866) but address a failing integration test. In short, when a client requests an invalid application or version, we need to include the errant application/version in the error message